.github/workflows/windows.yml

```diff
@@ -25,31 +25,26 @@ jobs:
     name: Try to download prebuilt OpenVINO
     outputs:
       status: ${{ steps.openvino_download.outcome }}
+      ov_package_path: ${{ steps.openvino_download.outputs.ov_package_path }}
+      ov_wheel_input: ${{ steps.openvino_download.outputs.ov_wheel_input }}
+      ov_wheel_source: ${{ steps.openvino_download.outputs.ov_wheel_source }}
     timeout-minutes: 10
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-20.04
-
-    steps:
-      - name: Download OpenVINO build
-        id: openvino_download
-        run: |
-          wget ${{ env.OV_TARBALL}} --progress=bar:force:noscroll -O openvino_package.zip
-          unzip -l openvino_package.zip
-        continue-on-error: true
-
-      #
-      # Upload to artifacts
-      #
-
-      - name: Upload openvino package
-        if: steps.openvino_download.outcome == 'success'
-        uses: actions/upload-artifact@v4
-        with:
-          name: openvino_package
-          path: openvino_package.zip
-          if-no-files-found: 'error'
+    runs-on: aks-linux-2-cores-8gb
+    container:
+      image: 'openvinogithubactions.azurecr.io/openvino_provider:0.1.0'
+      volumes: [ "/mount:/mount" ]
```

Comment:
do we need a docker image to run OV provider ?

Reply:
Just to avoid a separate step installing some tools for working with jsons (latest.json) or yamls (manifest.yml) etc. It looks like this: https://github.com/openvinotoolkit/githubactions_infra/blob/master/dockerfiles/openvino_provider/Dockerfile